### ISSUE TITLE
PHP error when running cron.php from cli

### ISSUE THREAD

**Issue by individual-it** [owncloud/core#18726]
I'm running OC on a bitnami VM.
Cron from Ajax or webcron work fine but not from cli/crontab

### Steps to reproduce
1. start bitnami VM with ownCloud 8.1.1
2. set "Use system's cron service to call the cron.php file every 15 minutes."
3. enable shell for "daemon" user in /etc/passwd
4. run

 ````
sudo -u bitnami /opt/bitnami/php/bin/php -f /opt/bitnami/apps/owncloud/htdocs/cron.php
 ````

### Expected behaviour
cron.php executes without errors

### Actual behaviour

````
sudo -u bitnami /opt/bitnami/php/bin/php -f /opt/bitnami/apps/owncloud/htdocs/cron.php
PHP Warning:  flock() expects parameter 1 to be resource, boolean given in /opt/bitnami/apps/owncloud/htdocs/lib/private/config.php on line 201
PHP Fatal error:  Call to a member function getSystemConfig() on a non-object in /opt/bitnami/apps/owncloud/htdocs/lib/private/log.php on line 59
````
### Server configuration
**Operating system**:
Linux linux 3.13.0-55-generic #94-Ubuntu SMP Thu Jun 18 00:27:10 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
Bitnami VM

**Web server:**
Server version: Apache/2.4.16 (Unix)
Server built:   Aug  7 2015 10:28:13
**Database:**
Ver 5.6.26 for linux-glibc2.5 on x86_64 
**PHP version:**
PHP 5.5.28
**ownCloud version:** (see ownCloud admin page)
8.1.1
**Updated from an older ownCloud or fresh install:**
fresh
**List of activated apps:**
Enabled:
 - files: 1.1.10
 - files_external: 0.2.3
 - files_locking: 
 - files_sharing: 0.6.2
 - firstrunwizard: 1.1
 - provisioning_api: 0.2
 - updater: 0.6
Disabled:
 - activity
 - encryption
 - external
 - files_pdfviewer
 - files_texteditor
 - files_trashbin
 - files_versions
 - files_videoviewer
 - gallery
 - templateeditor
 - user_external
 - user_ldap
 - user_webdavauth

**The content of config/config.php:**

```
<?php
$CONFIG = array (
  'instanceid' => 'oc25f6hxruxt',
  'passwordsalt' => 'xxxxxx',
  'secret' => 'xxxxx',
  'trusted_domains' =>
  array (
    0 => '10.49.35.124',
    1 => '127.0.0.1',
  ),
  'datadirectory' => '/opt/bitnami/apps/owncloud/data',
  'overwrite.cli.url' => 'http://localhost/owncloud',
  'dbtype' => 'mysql',
  'version' => '8.1.1.3',
  'dbname' => 'bitnami_owncloud',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'bn_owncloud',
  'dbpassword' => 'xxxxx',
  'logtimezone' => 'UTC',
  'installed' => true,
  'ldapIgnoreNamingRules' => false,
  'loglevel' => 0,
  'memcache.local' => '\OC\Memcache\APCu'
);

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
smb
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

#### Web server error log

```
[Tue Sep 01 10:12:16.973947 2015] [lbmethod_heartbeat:notice] [pid 27840:tid 140336205166400] AH02282: No slotmem from mod_heartmonitor
[Tue Sep 01 10:12:17.030566 2015] [mpm_event:notice] [pid 27840:tid 140336205166400] AH00489: Apache/2.4.16 (Unix) OpenSSL/1.0.1p configured -- resuming normal operations
[Tue Sep 01 10:12:17.030959 2015] [core:notice] [pid 27840:tid 140336205166400] AH00094: Command line: '/opt/bitnami/apache2/bin/httpd.bin -f /opt/bitnami/apache2/conf/httpd.conf'
```

#### ownCloud log (data/owncloud.log)
```
{"reqId":"kUH4H6kSV5s0VewD43sp","remoteAddr":"10.49.32.166","app":"PHP","message":"Array to string conversion at \/opt\/bitnami\/apps\/owncloud\/htdocs\/lib\/private\/template\/functions.php#36","level":3,"time":"2015-09-01T04:27:53+00:00"}
```

**Comment by PVince81** [comment]
@individual-it did you have any more luck with 8.1.3 ?

The flock error might be fixed there.

**Comment by individual-it** [comment]
yes, seems to be fixed in 8.1.3
closing this issue
